components/tooltip/test/tooltip-helper.js

```diff
@@ -1,4 +1,4 @@
-import { oneEvent } from '@brightspace-ui/visual-diff/helpers/index.js';
+import { oneEvent } from '@brightspace-ui/visual-diff/helpers';
```

Comment:
Is this because it _knows_ it's a package due to the file name being `index.js`?  Or is it just checking that it's not a relative path and `import {customElement} from 'lit/decorators'` would work as well?

Reply:
Hmm, good question. Looking [at the source](https://github.com/import-js/eslint-plugin-import/blob/master/src/rules/extensions.js#L160), it appears to just treat anything starting with `@something` or any characters (i.e. not a `../` relative path) as a module.

So... that's not good enough actually, because `import { ifDefined } from 'lit-html/directives/if-defined';` both doesn't get caught by the linter and also doesn't work in Lit 2.0.

Back to the drawing board I think. Like this change still catches _some_ of the issues, just not all of them. So I think we can leave it in the eslint config for now and fix these places in core... but we'll need something more.